### DISCUSSION TITLE
LB vs Alfred (question from @Rhydderch)

### DISCUSSION THREAD

**Discussion by Ptujec** [Ptujec/LaunchBar#29] [General]
Here is the question by @Rhydderch: 

> Hello,
> 
> First of all sorry to use this issue as a way to ask you this question but I am really curious about your use of launchbar. You really seem to be a power user. It is the first time I hear about it. Is it similar/better to Alfred for mac?
> 
> I mainly code, research, write (I am a PhD student), and use obsidian every day.

I felt here is a better place to answer this.

**Comment by Ptujec** [comment]
Yes, I use LaunchBar a lot. LaunchBar has a long history. [It's the first of this kind of application on the Mac](https://mastodon.nl/@mlinzner/109800786409145571). 

There is a lot of functionality that both Alfred and LaunchBar share. The difference is in the details. One example is that LaunchBar remembers the last thing you used. It also remembers the latest query of each search or action individually. Alfred does not do this. That is by design and some people like it and others don't. 
There are quite a few of those differences. Some will jump right at you if you are used to one of them and try to use the other.

I would recommend to [watch the videos on LaunchBars website that explain the main concepts and functionality](https://www.obdev.at/products/launchbar/videos.html). 

Regarding Obsidian. I made a [LaunchBar action for that](https://github.com/Ptujec/LaunchBar/tree/master/Obsidian-Actions#launchbar-actions-for-obsidian). But I don't really use Obsidian. So I don't know how helpful it actually is. It totally possible that you can find a better Alfred workflow or Raycast extension for Obsidian. I am not aware of other LaunchBar actions for Obsidian.

**Reply by Rhydderch** [reply]
Thank you for sharing your experience. I think I will have to try both and see which one I like most. Price seems similar too.

**Reply by Ptujec** [reply]
Raycast is another option which is very popular at the moment. It's VC funded and has a very long privacy page though.
